pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+Please include a summary of the changes this PR introduces for the codebase.
+Also specify if some sections need special attention, and why you want to introduce this change.
+
+Make sure to split changes across multiple pull requests, as we won't review bundled pull requests.
+
+Finally, make sure your PR follows our code of conduct before posting (Check our [contributing guidelines](CONTRIBUTING.md) if you're not sure).
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] :bug: Bug fix (Add `Fix #(issue)` to your PR)
+- [ ] :sparkles: New feature
+- [ ] :fire: Breaking change
+- [ ] :memo: Documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can run the tests. Please also list any relevant details for your test configuration:
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project (pep8 AND pyflakes)
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] If needed, I changed related parts of the documentation
+- [ ] I included tests in my PR
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] If my PR aims to fix an issue, I referenced it using `#(issue)`
```

Comment:
All good, though for a fledgling project, sometimes we might keep the process even simpler for better adoption. No need to change here, though.